console/v1alpha1/0000_10_consoleplugin.crd.yaml

```diff
@@ -17,7 +17,163 @@ spec:
     plural: consoleplugins
     singular: consoleplugin
   scope: Cluster
+  conversion:
+    strategy: Webhook
+    webhook:
+      conversionReviewVersions: ["v1", "v1alpha1"]
+      clientConfig:
+        service:
+          namespace: openshift-console-operator
+          name: conversion-webhook
+          path: /plugins-crd-convert
```

Comment:
@JoelSpeed not sure how we infest the caBundle for the service
https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#service-reference